packages/gatsby-plugin-mdx/README.md

```diff
@@ -664,41 +664,8 @@ However, we'd recommend placing such files into the `src/pages` directory and `g
 
 1. Instead of querying for the `body` on the MDX node, the page template now receives the transformed MDX as `children` property.
 1. You no longer need to use `<MDXRenderer>` as you can use `{children}` directly.
-1. If you have given your query a name (e.g. `query PostTemplate`) you have to remove the name. Gatsby automatically creates a name internally, but if a name is provided you'll see a "Duplicate query" warning.
 
-```diff
```

Comment:
You removed too much here. It still has to show the `children` diff noted in the first bullet point.

You also shouldn't change this to four backticks, you've made the whole section below a code block